share/picongpu/benchmarks/TWEAC-FOM/include/picongpu/param/incidentField.param

```diff
@@ -56,6 +56,7 @@
 #pragma once
 
 #include "picongpu/fields/background/templates/twtstight/twtstight.hpp"
+#include "picongpu/fields/background/templates/twtstight/twtstight.tpp"
```

Comment:
This will be fixed in https://github.com/ComputationalRadiationPhysics/picongpu/pull/5155
If you include this in param you create cyclic includes. If this change would solve the problem we could simply include the top in the hpp. This is not the case here.

Reply:
Done.